builder/symlink.go

```
@@ -76,7 +79,21 @@ func main() {
 				}
 
 				for _, f := range files {
-					os.Symlink(filepath.Join(src, f.Name()), filepath.Join("vendor", path, f.Name()))
+					innerSrc := filepath.Join(src, f.Name())
+					dst := filepath.Join("vendor", path, f.Name())
+					if err := os.Symlink(innerSrc, dst); err != nil {
+						// assume it's an existing directory, try to link the directory content instead.
+						// TODO should we do this recursively
+						files, err := ioutil.ReadDir(innerSrc)
+						if err != nil {
+							panic(err)
+						}
+						for _, f := range files {
+							if err := os.Symlink(filepath.Join(innerSrc, f.Name()), filepath.Join(dst, f.Name())); err != nil {
+								fmt.Println("ignore symlink error", filepath.Join(innerSrc, f.Name()), filepath.Join(dst, f.Name()))
```

```suggestion
```
this log is noisy too, not sure if we should just ignore the error.